_posts/2018-04-10-system-information-with-rust.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: System Information with Rust
+date: 2018-04-10
+categories:
+    - projects
+description: A Rust library that returns information about the local system it's running on.
```

Comment:
could you add an image attribute w/ a screenshot of the project? Should be in the form:
image: https://urlgoes.here/